### ISSUE TITLE
[SEV2] Visual label and aria label are different for the 'Auto' menu item under 'Choose refresh time interval' button

### ISSUE THREAD

**Issue by JonCole** [grafana/grafana#116511]
**Pre-Requisite**:

Achieve viewport 320 X 256 by adjusting the browser. (Open developer's tool, resize and zoom the browser to achieve viewport 320 X 256).

**Steps to reproduce**:
1. Create a new dashboard
2. add a new visualization
3. Navigate to 'Change refresh time interval' and expand it.
4. Verified whether visual label and aria label are different for the 'Auto' menu item or not.

**Actual Result**:
Visual label and aria label are different for the 'Auto' menu item under 'Change refresh time interval' present under 'New dashboard' screen.

**Expected Result**
Visual label and aria label should be same for the 'Auto' menu item under 'Change refresh time interval' present under 'New dashboard' screen.

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

WCAG: [2.5.3 Label in Name](https://liquid.microsoft.com/Web/Object/Read/ms.accessibility/Requirements/2.5.3)
EN 301 549: [PDF of EN 301 549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf), which applies WCAG principles for Label in Name to 9.2.5.3 web, 10.2.5.3 non-web documents, and 11.2.5.3.1 software.

**Environment**:
- Grafana version: 12.2.0
- Data source type & version: n/a
- User OS & Browser: any
- Others:

**Comment by JonCole** [comment]
We consider this a Sev 2 and ask that it be fixed within 60 days
